ansible_runner/runner.py

```diff
@@ -137,8 +138,8 @@ def run(self):
         self.last_stdout_update = time.time()
         try:
             child = pexpect.spawn(
-                self.config.command[0],
-                self.config.command[1:],
+                command[0],
```

Comment:
Are we sure that the thing with unicode characters in the environment causing `spawn()` to hang doesn't also apply to the command?